science/ViennaRNA/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+name                ViennaRNA
+version             2.4.14
+categories          science
+platforms           darwin
+maintainers         @GiovanniBussi openmaintainer
+description         C Library and programs for the prediction and comparison\
+                    of RNA secondary structures.
+long_description    The Vienna RNA Package consists of a C code library and\
+                    several stand-alone programs for the prediction and\
+                    comparison of RNA secondary structures.
+homepage            https://www.tbi.univie.ac.at/RNA/
+master_sites        http://www.tbi.univie.ac.at/RNA/download/sourcecode/2_4_x/ViennaRNA-${version}.tar.gz
```

Comment:
master_sites should be the path to the directory containing the file, not including the filename itself.

To avoid hardcoding the branch here, define a [branch variable](https://trac.macports.org/wiki/PortfileRecipes#branch).

```
set branch          [join [lrange [split ${version} .] 0 1] _]_x
master_sites        https://www.tbi.univie.ac.at/RNA/download/sourcecode/${branch}/
```

Reply:
Better yet, don't repeat the homepage; use the variable:

```
set branch          [join [lrange [split ${version} .] 0 1] _]_x
master_sites        ${homepage}download/sourcecode/${branch}/
```

Reply:
Thanks! I implemented it